framework/scheduler.go

```diff
@@ -98,6 +101,17 @@ func (s *StackDeployScheduler) Start() error {
 		}
 	}()
 
+	s.Cron.Start()
+	go func() {
```

Comment:
Ok but let's make sure we remove this when schedule management logic is implemented